zanata-war/src/main/java/org/zanata/action/ProjectSlug.java

```diff
@@ -22,16 +22,18 @@
 
 import java.io.Serializable;
 
-import org.apache.deltaspike.core.api.scope.ViewAccessScoped;
+import org.apache.deltaspike.core.api.scope.ConversationGroup;
+import org.apache.deltaspike.core.api.scope.GroupedConversationScoped;
 
 /**
- * We set this to ViewAccessScoped so that it can retain the value accross
- * requests (original request, subsequent ajax requests)
+ * We set this to GroupedConversationScoped so that it can retain the value
+ * across requests (original request, subsequent ajax requests)
```

Comment:
Yes... maybe if only to remind us of why we are relying on conversation scope when in theory view socpe should to the trick. (i.e. using our autocompletes and filters)